### PR TITLE
add accessibility considerations

### LINKED ISSUE #411: No references to accessibility
Inaccessible permissions are a huge issue.  They can make people with disabilities less safe, while not even KNOWING they are less safe.  I believe it is important to have a section in this document specifically on accessibility cross-linked to the relevant WCAG criteria.  Off the top of my mind, that would be labels, meaningful sequence, keyboard interaction, non-graphical descriptions, keyboard focus indicator, and color ratio.  There may be a few I missed.  Happy to attend a meeting to discuss further. You can reach out to me at sbyrnehaber@vmware.com to schedule that.

### PR DESCRIPTION
**Pull request by chaals**
closes #411 (minimally - this is still punting issues like actually explaining to real people what "this feature on this origin" actually means).

The following tasks have been completed:

 * [ ] Modified Web platform tests (link)

Implementation commitment:

 * [ ] WebKit (link to issue)
 * [ ] Blink (link to issue)
 * [ ] Gecko (link to issue)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/permissions/pull/412.html#accessibility-considerations" title="Last updated on Jan 25, 2023, 7:32 PM UTC (5602470)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/permissions/412/2c61163...5602470.html" title="Last updated on Jan 25, 2023, 7:32 PM UTC (5602470)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1309,13 +1309,24 @@ <h2 id="privacy-considerations">
...
       </p>
     </section>
+    <section id="accessibility-considerations">
+      <h2>
+        Accessibility considerations
+      </h2>
+      <p>
+        Interfaces intended to allow the user to  review, update, and reset the
+        [=permission=] [=permission/state=] of [=powerful features=] associated with an [=origin=].
+        MUST respect platform accessibility conventions.
```
Comment (jyasskin) [inline index.html:1319]: Isn't this true of every UA interface? Why would permissions need special text for this? Were there specific problems with permission UI? Did browsers push back when you filed bugs about those?

Comment (miketaylr) [reply index.html:1319]: ping @chaals

Comment (marcoscaceres) [reply index.html:1319]: I tend to agree with @jyasskin... this is seems somewhat overly general and doesn't really much. 

I'd be inclined to add guidance around this if we had seen issues (or foresee accessibility issues!) with the UIs being presented. 

So far, the UIs have been ok, no?

**Comment by marcoscaceres** [comment]
We discussed this in the Editors meeting and decided we will keep this as is. Hope that's ok @chaals.

**Pull request closed without merging**